docs/advanced/using-standalone.md

```diff
@@ -10,7 +10,7 @@ applications.
 
 ## LayerManager class
 
-The deck.gl `LayerManager` class handles updates, drawing and picking
+The deck.gl [LayerManager](/src/lib/layer-manager.js) class handles updates, drawing and picking
```

Comment:
We now have a place holder for the `LayerManager` class in the API reference. Better to link to that, and then have that doc link to the src.